Formula/htmlq.rb

```diff
@@ -0,0 +1,18 @@
+class Htmlq < Formula
+  desc "Uses CSS selectors to extract bits content from HTML files"
+  homepage "https://github.com/mgdm/htmlq"
+  url "https://github.com/mgdm/htmlq/archive/0cb3b720451f85710b6c68894c39d521ca0d9b15.tar.gz"
+  sha256 "13bce9d1620bc8abd8fa5e408d76e67bc33b86d2640dc8753b6a8e119739b2aa"
+  version "0.1.1"
+  license "MIT"
+  head "https://github.com/mgdm/htmlq.git", branch: "master", revision: "0cb3b720451f85710b6c68894c39d521ca0d9b15"
```

Comment:
```suggestion
  head "https://github.com/mgdm/htmlq.git", branch: "master"
```
We don't specify a `revision` in `head`.

Reply:
`revision` removed